code/modules/projectiles/guns/energy/netgun_yw.dm

```diff
@@ -0,0 +1,36 @@
+
+/obj/item/weapon/gun/energy/hunter
+	name = "Hybrid 'Hunter' net gun"
+	desc = "A Hephaestus-designed hybrid of a taser and the energy net gun, usually dubbed 'Hunter' stunner and energy net launcher, \
+			for when you want criminals to stop acting like they're on a 20th century British comedy sketch show."
+	catalogue_data = list(/datum/category_item/catalogue/information/organization/hephaestus)
+	icon = 'icons/obj/gun_vr.dmi'
+	icon_state = "hunter"
+	item_state = "gun" // Placeholder
+	mode_name = "stun"
+
+	fire_sound = 'sound/weapons/eluger.ogg'
+	origin_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 6, TECH_MAGNET = 4)
+	projectile_type = /obj/item/projectile/beam/stun/blue
+	charge_cost = 240
+	fire_delay = 5
+
+	firemodes = list(
+		list(mode_name="stun", projectile_type=/obj/item/projectile/beam/stun/blue, fire_sound='sound/weapons/Taser.ogg', charge_cost=240, fire_delay=5),
```

Comment:
```suggestion
		list(mode_name="stun", projectile_type=/obj/item/projectile/beam/stun/blue, fire_sound='sound/weapons/Taser.ogg', charge_cost=480, fire_delay=5),
```
Makes it have a drawback instead of just being a taser with a netgun mode slapped ontop (5 taser shots)

Reply:
I was thinking on increasing fire delay?, taser normally has 15 shots so this one has 10,  but can bw nerfed ye

Reply:
if you wanna do the math you can bump it up to 7 shots at most tho fire delay will have to be increased

Reply:
320 seems good, its 7,5 making exactly the half a taser, when used with hypers its 15, like a taser with normao cells, the fire delay ill do some tests

Reply:
sounds good

Reply:
the code i removed was needed, without it it shot laser before i changed modes, changed cost to 320 and delay to 10

Reply:
Ahh